bears/python/MypyBear.py

```diff
@@ -0,0 +1,157 @@
+from collections import namedtuple
+import textwrap
+import sys
+import re
+import os
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+OUTPUT_REGEX = (r'(?P<filename>[^:]+):((?P<line>\d+):)? '
+                '(?P<severity>[^:]+): (?P<message>.*)')
```

Comment:
this is defined very far away from it's usage, also it doesn't need to be public, can we at least put it into the class?